.github/workflows/sdk.yml

```diff
@@ -24,6 +24,13 @@ jobs:
           path: 'sdk'
       - name: Copy API definition
         run: cp service/src/openapi.json sdk/openapi.json
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
```

Comment:
I think it's OK to build with Java 17 (or 21 for that matter) so long as we support 11 runtimes. That's more a matter for the sdk repo, though.